README.md

```diff
@@ -1,3 +1,44 @@
 # TheSportsDB
 GraphQL wrapper for TheSportsDB
 API doc - https://www.thesportsdb.com/api.php
+
+## How to use
+
+### Execute loal server
+
+Two ways for execute local server
+
+#### Using [Visual Studio Code](https://azure.microsoft.com/en-us/products/visual-studio-code/)
+
+Modify configurations of `launch.json`.
+
+```json:.vscode/launch.json
+{
+    ...
+    "configurations": [
+        {
+            ...
+            "runtimeExecutable": null, // Write your node path. (run: 'which node' on terminal)
+            ...
+    ]
+}
+```
+
+Press `F5` for run debugger.
+
+#### Using CLI
+
+Type this command
+
+```bash
+cd server
+npm run local
+```
+
+### Execute query
+
+We are able to install [GraphQL Playground](https://github.com/prisma-labs/graphql-playground) for execute query.
```

Comment:
There is no need for the install. Running the server and pointing the browser to localhost:4000 will bring it up. It is part of Apollo.

Reply:
I didn't know there was such a function. Fixed in [67d2e2f](https://github.com/jsBlackBelt/TheSportsDB/pull/2/commits/67d2e2f5506ea82bd137cb35bdd5e9f008303969)